graylog2-server/src/main/java/org/graylog/plugins/views/search/rest/ViewSharingResource.java

```diff
@@ -45,7 +45,7 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
-@Api(value = "Enterprise/Views", description = "View Sharing management")
+@Api(value = "Views")
```

Comment:
I think this should be more precise, maybe "Views Sharing"?

Reply:
Good point! Will do shortly.

Reply:
Changed it to "Views/Sharing".